layers/meta-balena-allwinner/conf/samples/bblayers.conf.sample

```diff
@@ -14,7 +14,9 @@ BBLAYERS ?= " \
     ${TOPDIR}/../layers/meta-openembedded/meta-python \
     ${TOPDIR}/../layers/meta-sunxi \
     ${TOPDIR}/../layers/meta-balena/meta-resin-common \
-    ${TOPDIR}/../layers/meta-balena/meta-resin-sumo \
+    ${TOPDIR}/../layers/meta-balena/meta-balena-warrior \
     ${TOPDIR}/../layers/meta-balena-allwinner \
     ${TOPDIR}/../layers/meta-rust \
     "
+
+BBMASK += "${TOPDIR}/../layers/meta-sunxi/recipes-bsp/u-boot/u-boot_%.bbappend"
```

Comment:
Why this change? We need to have it detailed in the git log why this was done

Reply:
The masked file, if unmasked, applies to u-boot recipes from `meta-balena-allwiner` and this is not necessary as all the recipes files have proper/custom bbappends according to the u-boot version they refer to.
I will update the git log to reflect this.

Reply:
I didn't notice that meta-sunxi was still on on `master`, I checked out to `warrior` and the file is gone. Updated the patchset to reflect this.